engine/src/test/java/com/redhat/cloud/notifications/SecuredEmailTemplatesInDbHelper.java

```diff
@@ -0,0 +1,101 @@
+package com.redhat.cloud.notifications;
+
+import com.redhat.cloud.notifications.config.FeatureFlipper;
+import com.redhat.cloud.notifications.db.ResourceHelpers;
+import com.redhat.cloud.notifications.db.StatelessSessionFactory;
+import com.redhat.cloud.notifications.db.repositories.TemplateRepository;
+import com.redhat.cloud.notifications.ingress.Action;
+import com.redhat.cloud.notifications.models.Bundle;
+import com.redhat.cloud.notifications.templates.EmailTemplateMigrationService;
+import com.redhat.cloud.notifications.templates.TemplateService;
+import com.redhat.cloud.notifications.templates.models.Environment;
+import io.quarkus.qute.TemplateInstance;
+import io.quarkus.test.junit.mockito.InjectSpy;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import javax.inject.Inject;
+import javax.persistence.NoResultException;
+import java.util.Map;
+
+public abstract class SecuredEmailTemplatesInDbHelper {
+
+    protected static final String BUNDLE_RHEL = "rhel";
+
+    protected static final String COMMON_SECURED_LABEL_CHECK = "common template for secured env";
+
+    private static final boolean SHOULD_WRITE_ON_FILE_FOR_DEBUG = false;
+
+    @Inject
+    Environment environment;
+
+    @Inject
+    protected ResourceHelpers resourceHelpers;
+
+    @Inject
+    protected StatelessSessionFactory statelessSessionFactory;
+
+    @InjectSpy
+    protected TemplateRepository templateRepository;
+
+    @Inject
+    protected TemplateService templateService;
+
+    @Inject
+    private FeatureFlipper featureFlipper;
```

Comment:
Quarkus doesn't like when something `private` is injected. It works but Quarkus complains in the logs:
```
2023-03-10T14:21:18.6616524Z 2023-03-10 14:21:18,647 INFO  [io.qua.arc.pro.BeanProcessor] (build-13) Found unrecommended usage of private members (use package-private instead) in application beans:
2023-03-10T14:21:18.6617591Z 	- @Inject field com.redhat.cloud.notifications.SecuredEmailTemplatesInDbHelper#featureFlipper,
2023-03-10T14:21:18.6618466Z 	- @Inject field com.redhat.cloud.notifications.SecuredEmailTemplatesInDbHelper#featureFlipper,
2023-03-10T14:21:18.6619324Z 	- @Inject field com.redhat.cloud.notifications.SecuredEmailTemplatesInDbHelper#featureFlipper
2023-03-10T14:21:18.6620065Z 	- and 6 more - please enable debug logging to see the full list
```
```suggestion
    FeatureFlipper featureFlipper;
```

Reply:
Hum ok, good to know :)

Reply:
FWIW this seems to be the reason why it doesn't like it: https://quarkus.io/guides/cdi-reference#native-executables-and-private-members